docs/components/FormField/Mobile.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { FormField, InputText, Text } from "@jobber/components-native";
+
+export default {
+  title: "Components/Private/FormField/Mobile",
+  component: FormField,
+  parameters: {
+    viewMode: "story",
+    previewTabs: { code: { hidden: false } },
+    viewport: { defaultViewport: "mobile1" },
+    showNativeOnWebDisclaimer: true,
+  },
+} as ComponentMeta<typeof FormField>;
+
+const BasicTemplate: ComponentStory<typeof FormField> = args => {
+  return (
+    <FormField {...args}>
+      {field => {
+        return <InputText value={field.value} placeholder="Enter name here" />;
+      }}
+    </FormField>
+  );
+};
+
+const WithValidationsTemplate: ComponentStory<typeof FormField> = args => {
+  return (
+    <FormField
+      {...args}
+      validations={{
+        maxLength: {
+          value: 5,
+          message: "Should not exceed 5 characters",
+        },
+      }}
+    >
+      {(field, error) => {
+        return (
+          <>
+            {error && <Text variation="error">{error.message}</Text>}
+            <InputText
+              value={field.value}
+              placeholder="Should not exceed 5 characters"
+            />
+          </>
+        );
+      }}
+    </FormField>
+  );
+};
```

Comment:
In order to make this example work, you'll need to wrap it in a `FormProvider` and call the `onChange` callback from FormField:

```suggestion
const WithValidationsTemplate: ComponentStory<typeof FormField> = args => {
  const methods = useForm({ mode: "onChange" });
  return (
    <FormProvider {...methods}>
      <FormField
      {...args}
      validations={{
        maxLength: {
          value: 5,
          message: "Should not exceed 5 characters",
        },
      }}
    >
      {(field, error) => {
        return (
          <>
            {error && <Text variation="error">{error.message}</Text>}
            <InputText
              value={field.value}
              placeholder="Should not exceed 5 characters"
              onChangeText={field.onChange}
            />
          </>
        );
      }}
    </FormField>
  );
};
```

Reply:
Don't use this suggestion to commit, the formatting is all wrong 😂

Reply:
thank you!